CHANGELOG.md

```diff
@@ -2,8 +2,11 @@
 
 ## v2.4dev
 
+This patch release to removes the Graphviz dependency from default pipeline template.
+
```

Comment:
```suggestion
```
I think info in the `## Template` are enough

Reply:
In the current form it is quite redundant, but this change might need some visibility. Not just because it removes the dependency, but because it should probably mention in the more prominent sentence that the default DAG graph format has changed and that user configs need to change to keep getting the SVG output.

Reply:
ok for me then

Reply:
I added a sentence clarifying the change.

Reply:
Please double check if it makes sense. I am not yet familiar enough with nextflow to judge what I am writing about configs ...

Reply:
After that, if you think it's ready to merge, I am all for it.